### PR TITLE
refactor: use styled-components instead of emotion

### PR DESCRIPTION
**Pull request by walker-tx**
Replaced any reference of emotion to styled-comonents. This included all
components, gatsby-config, and npm package files.

### CODE REVIEW

**Comment by walker-tx** [comment]
Just realized that I forgot to change the README.md

File: src/pages/index.jsx [hunk 1]
```diff
@@ -17,7 +17,7 @@ import favicon from '../favicon.png';
 import rightArrow from '../right-arrow.svg';
 import github from '../github.svg';
 
-injectGlobal`
+createGlobalStyle`
```
Comment (LekoArts) [inline src/pages/index.jsx]: You need to create a component and use it in the Index React component:
https://www.styled-components.com/docs/api#createglobalstyle

**Review by LekoArts — CHANGES_REQUESTED** [review]
Hey, sorry for the late response! Been very busy. There's just one thing that needs to be changed

File: src/pages/index.jsx (hunk 1 above)
Comment (walker-tx) [reply src/pages/index.jsx]: How careless of me! I'll make the correction.

**Comment by LekoArts** [comment]
And please also remove the `package-lock.json` and use `yarn` to update the `yarn.lock` file 👍

**Comment by walker-tx** [comment]
The requested changes have been made. Sorry about the yarn.lock thing. Haven't messed with yarn really so didn't pick up on that. Learned some new things today!

**Review by LekoArts — APPROVED** [review]

**Comment by LekoArts** [comment]
Thanks!

**Comment by walker-tx** [comment]
huzzah!

**Pull request merged**